Engine/Module/ModuleLight.cpp

```diff
@@ -54,6 +86,9 @@ void ModuleLight::RenderDirectionalLight(const float3& mesh_position)
 
 		size_t light_direction_offset = App->program->uniform_buffer.lights_uniform_offset + 4 * sizeof(float);
 		glBufferSubData(GL_UNIFORM_BUFFER, light_direction_offset, sizeof(float3), light->owner->transform.GetFrontVector().ptr());
+	
+		directional_light_rotation = directional_light_rotation.LookAt(float3::unitZ, light->owner->transform.GetFrontVector(), float3::unitY, light->owner->transform.GetUpVector());
```

Comment:
Isn't this the same as `directional_light_rotation = light->owner->transform.GetFrontVector()` ?

Reply:
Probably (?)